lib/open_project/configuration/helpers.rb

```diff
@@ -77,6 +77,18 @@ def available_file_uploaders
         }
       end
 
+      # overrides default getter in OpenProject::Configuration
+      def hidden_menu_items
+        menus = self['hidden_menu_items'].map do |label, nodes|
```

Comment:
Unused block argument - `label`. If it's necessary, use `_` or `_label` as an argument name to indicate that it won't be used.